components/composed/model/ModelPagination/ModelPagination.tsx

```diff
@@ -4,16 +4,23 @@ import { graphql, useFragment } from "react-relay";
 import { ModelPaginationFragment$key } from "@/relay/ModelPaginationFragment.graphql";
 
 interface ModelPaginationProps<T extends ModelPaginationFragment$key> {
-  data: T;
+  data?: T | null;
 }
 
 function ModelPagination<T extends ModelPaginationFragment$key>({
   data,
 }: ModelPaginationProps<T>) {
-  const { pageInfo } = useFragment<ModelPaginationFragment$key>(fragment, data);
+  const enhancedData = useFragment<ModelPaginationFragment$key>(
+    fragment,
+    data || null
+  );
+  if (!enhancedData || !enhancedData.pageInfo) return null;
```

Comment:
We should also return null if there's only one page - not a blocker for this PR, just making a note